### PR TITLE
added space to header to fix overrun issue

### PR DESCRIPTION
**Pull request by zr-msft**
fixes #223

Added a space to path in title to fix overrun issue. 

This provides an alternate solution to #255

### CODE REVIEW

**Review by FeynmanZhou — APPROVED** [review]
LGTM

File: content/en/docs/troubleshooting.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ In addition, there is `--debug`, which is intended for developers to debug the n
 
 ## When I verify an artifact, I get the error 'Error: open $HOME/.config/notation/trustpolicy.json: no such file or directory'
 
-This error is likely related to trust policy configuration. Verify you have a trust policy set up before you attempt to verify an artifact. `notation policy show` can be used to view the current trust policy in use. If you need to override an existing trust policy with a new one, you can use the `notation policy import` command to import a new trust policy. For more details, see [Manage trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}})
+This error is likely related to trust policy configuration. Verify you have a trust policy set up before you attempt to verify an artifact. `notation policy show` can be used to view then current trust policy in use. If you need to override an existing trust policy with a new one, you can use the `notation policy import` command to import a new trust policy. For more details, see [Manage trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}})
```
Comment (yizha1) [inline content/en/docs/troubleshooting.md:91]: I think `the` is correct.

Comment (zr-msft) [reply content/en/docs/troubleshooting.md]: fixed

**Review by yizha1 — DISMISSED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
@zr-msft Should we fix the overrun issue on the front end? If so, I guess @sanjayk0508 could help.

**Comment by zr-msft** [comment]
> @zr-msft Should we fix the overrun issue on the front end? If so, I guess @sanjayk0508 could help.

This was related to a file path that was too long. I'm open to @sanjayk0508 taking a look, but this PR solves the issue for now

**Comment by yizha1** [comment]
@zr-msft Would you mind resolving the conflicts? Thanks.

**Comment by zr-msft** [comment]
@FeynmanZhou would you mind taking a look so i can merge?

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**